### PR TITLE
update with new activities

### PR DESCRIPTION
**Pull request by vix633**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2385,7 +2385,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#657cf8d3475b895a81679c10f74797267e9ef9cb",
+      "version": "github:BrightspaceHypermediaComponents/activities#cbfe7aafc232282fc3f94a880838b8810b94981c",
```
Comment (mtjandra) [inline package-lock.json]: Don't you want `047a606232a986bfb2de117208f8dc7d9b56dc82` ?

Comment (svanherk) [inline package-lock.json]: This is not including your latest change

Comment (vix633) [reply package-lock.json]: Yeah, FYI this was a byproduct of me me creating version `v1.15` rather than `v1.15.0`

Comment (vix633) [reply package-lock.json]: Yeah i wanted to drill down into why that was.

Comment (svanherk) [reply package-lock.json]: Cool, you'll have to rerun and push your new changes to get 1.15.0

**Comment by mtjandra** [comment]
I just tagged a new release if you wouldn't mind using the latest https://github.com/BrightspaceHypermediaComponents/activities/releases/tag/v1.16.0

**Pull request closed without merging**